doc/controls/ShadowContainer.md

```diff
@@ -11,6 +11,19 @@ A reference to [Uno.Toolkit.Skia.WinUI](https://www.nuget.org/packages/Uno.Toolk
 ## Remarks
 Currently, the control simply mimics the shape of its content by size and by corner radius. Complicated shapes, such as text or pictures with alpha, are not yet supported.
 
+In order for inset (inner) shadows to be paint on top of background, the `ShadowContainer`'s child element should not have a `.Background` or `.Fill`(for `Shape`s), it should be omitted or set to transparent. And, the background/fill value should be instead set on the `ShadowContainer` itself.
```

Comment:
Should we put this in a `[!WARNING]` or `[!NOTE]`?

Reply:
Yes a [!WARNING] would be best I think or [!NOTE] at least